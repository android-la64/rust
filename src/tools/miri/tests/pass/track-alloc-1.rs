// Ensure that tracking early allocations doesn't ICE Miri.
// Early allocations are probably part of the runtime and therefore uninteresting, but they
// shouldn't cause a crash.
// compile-flags: -Zmiri-track-alloc-id=1
fn main() {}
