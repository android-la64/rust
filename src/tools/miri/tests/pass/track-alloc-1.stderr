note: tracking was triggered
   |
   = note: created allocation with id 1
   = note: (no span available)

